### PR TITLE
News Page

### PR DESCRIPTION
**Pull request by ngthanhvinh**

### CODE REVIEW

**Review by guruprerana — DISMISSED** [review]

**Comment by ngthanhvinh** [comment]
I've resolved conflicts and merged with `master` and Khanh's `smallfix`. Ready to be merged now!

**Comment by ngkan** [comment]
Personally I don't like putting different kinds of helper functions into one big namespace.

File: src/trading-platform-pp.pro [hunk 1]
```diff
@@ -21,37 +21,48 @@ win32 {
 
 SOURCES += \
     api/api.cpp \
+    components/news.cpp \
```
Comment (ngkan) [inline src/trading-platform-pp.pro:24]: We don't need these since they have been renamed.

**Comment by ngkan** [comment]
There is a noticeable increase of start-up time, I guess this is due to calling the APIs.

**Comment by ngkan** [comment]
I can't get the news page to work correctly on Windows, but it should be fine on Linux (probably due to linking libcurl incorrectly).

LGTM nonetheless.

**Review by ngkan — APPROVED** [review]

**Comment by ngthanhvinh** [comment]
Regarding the namespace, I personally think that it is more convenient because we do not have too many helper functions, and separating them into smaller files does not improve the readability of the code. Also, I encountered some issues when working with the old qjsonobject helper file because there is no source file, so I decided to put all of the functions into a single helper namespace.

**Review by guruprerana — APPROVED** [review]

**Pull request merged**